contracts/Gauge.sol

```diff
@@ -687,27 +823,28 @@ contract Gauge is BaseGauge, IGauge {
         override
         returns (bool)
     {
-        return
-            _token != address(rewardToken) && _token != address(stakingToken);
+        return _token != address(rewardToken) && _token != address(asset);
     }
 
     /**
     @notice Kick `addr` for abusing their boost
-    @param _account Address to kick
+    @param _accounts Addresses to kick
     */
-    function kick(address _account) external updateReward(_account) {
-        Balance storage balance = _balances[_account];
+    function kick(address[] calldata _accounts) public {
+        for (uint256 i = 0; i < _accounts.length; ++i) {
+            _kick(_accounts[i]);
+        }
+    }
 
+    function _kick(address _account) internal updateReward(_account) {
+        uint256 balance = balanceOf(_account);
         require(
-            balance.boostedBalance >
-                (balance.realBalance * boostingFactor) / BOOST_DENOMINATOR,
+            _boostedBalances[_account] >
```

Comment:
Can this logic be a view function with address argument that return true or false? if i want to check accounts that potentially are candidates for kicking?

Reply:
They all are since quickly after a lock/deposit your veYFI token decreases.

Reply:
Dont understand your answer here regarding my question, whether this can be a view helper method we can check off chain or not.